src/static/helpers/slasHelper.ts

```diff
@@ -108,10 +108,13 @@ export async function authorize(
 ): Promise<{code: string; url: string; usid: string}> {
   const codeChallenge = await generateCodeChallenge(codeVerifier);
 
+  // Create a copy to override specific fetchOptions
+  const slasClientCopy = new ShopperLogin(slasClient.clientConfig);
+
   // set manual redirect on server since node allows access to the location
   // header and it skips the extra call. In the browser, only the default
   // follow setting allows us to get the url.
-  const fetchOptions = {
+  slasClientCopy.clientConfig.fetchOptions = {
     ...slasClient.clientConfig.fetchOptions,
     ...(!onClient && {redirect: 'manual'}),
```

Comment:
```suggestion
    redirect: onClient ? 'follow' : 'manual',
```
This is safer, because it guarantees the right redirect setting in both node and browser, rather than just in node. (Also I think it looks cleaner.)

Another option, while I'm here, is to use spread in the constructor so you only need one statement:
```typescript
  const slasClientCopy = new ShopperLogin({
    ...slasClient.clientConfig,
    fetchOptions: {
      ...slasClient.clientConfig.fetchOptions,
      redirect: onClient ? 'follow' : 'manual'
  });
```
Not sure which is cleaner.

Reply:
Leaning towards keeping the code as multiple statements. Seems a little easier to read especially with the comments